docs/contracts/functions.rst

```diff
@@ -318,6 +318,7 @@ will consume more gas than the 2300 gas stipend:
     you have to implement a receive Ether function (using payable fallback functions for receiving Ether is
     not recommended, since the fallback is invoked and would not fail for interface confusions
     on the part of the sender).
+    Note that ``send`` and ``transfer`` are scheduled to be deprecated in the next breaking version (0.9).
```

Comment:
Don't we deprecate them sooner, with this PR, and then with 0.9 (or another future breaking release) they are removed?

Reply:
I am not sure. I tried to find somewhere whether we define when it will be deprecated but couldn't locate anything yet.

Reply:
I guess I was using deprecation instead of removal. I will change the wording here to reflect that.